rmgpy/data/kinetics/familyTest.py

```diff
@@ -637,7 +638,13 @@ def testSaveFamily(self):
                                  os.path.join(settings['test_data.directory'], 'testing_database/kinetics/families/intra_H_copy/training/dictionary.txt')))
         finally:
             shutil.rmtree(os.path.join(settings['test_data.directory'], 'testing_database/kinetics/families/intra_H_copy'))
-
+    
+    def testReactantNumID(self):
+        family = self.database.families['R_Recombination']
+        spc = Molecule().fromSMILES("[CH2]CC[CH2]")
+        out = family._KineticsFamily__generateReactions(reactants=[spc],forward=True)
+        logging.error(out)
```

Comment:
I don't think you need to log this. If the assertion fails, it will print out the value.

Reply:
done